tests/test_updater_with_simulator.py

```diff
@@ -71,6 +79,33 @@ def test_refresh(self):
 
         self._run_refresh()
 
+    def test_targets(self):
+        # target does not exist yet
+        updater = self._run_refresh()
+        self.assertIsNone(updater.get_one_valid_targetinfo("file"))
+
+        self.sim.targets.version += 1
+        self.sim.add_target("targets", b"content", "file")
+        self.sim.update_snapshot()
+
+        # target now exists, is not in cache yet
+        updater = self._run_refresh()
+        info = updater.get_one_valid_targetinfo("file")
```

Comment:
`info` sounds a little ambiguous.
Maybe `file_info`?
So when we add more targets they will be `file2`, `file3` and when we get their info with `get_one_valid_targetinfo()` their corresponding variables will be `file2_info`, `file3_info`, etc.

Reply:
yeah... I'm trying to avoid long lines in test code but you may be right: the issue in these lines is `get_one_valid_targetinfo` not the variable name